### PR TITLE
feat(a380x/fws): Add V1 callout

### LINKED ISSUE #8577: [A380X] V1 callout
### Aircraft Version

Development

### Description

Not sure about the A320 family, but A380 has auto V1 call-out. Would be nice to add this feature. Doing so will also enhance realism in my opinion.

### References (optional)

https://youtu.be/fLrZloqULnY?si=xUwpaNJCPbx9Oidc
You can hear the V1 call-out by the synthetic voice when taking off

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by flogross89**
Fixes #8577 

## Summary of Changes

Adds the V1 callout to the FWS

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Perform cockpit preparation and FMS programming as per SOP, 
1) Take-off, V1 callout should be audible slightly before V1. Reject take-off after V1
2) Take-off once again, confirm that V1 callout sounds again

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts [hunk 1]
```diff
@@ -2284,6 +2287,19 @@ export class FwsCore {
...
+    const v1Threshold = v1 - 4;
+    this.v1SpeedConfirmNode.write(
+      v1 &&
+        (this.adr1Cas.get().valueOr(0) > v1Threshold ||
+          this.adr2Cas.get().valueOr(0) > v1Threshold ||
+          this.adr3Cas.get().valueOr(0) > v1Threshold),
+      deltaTime,
+    );
+    const v1CalloutCondition = this.fwcFlightPhase.get() === 4 && this.v1SpeedConfirmNode.read();
+    SimVar.SetSimVarValue('L:A32NX_AUDIO_V1_CALLOUT', SimVarValueType.Bool, v1CalloutCondition);
```
Comment (BlueberryKing) [inline fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts:2301]: Ideally, we wouldn't set this every iteration of the update loop and only once when the condition is met.

Comment (flogross89) [reply fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts:2301]: Do you mean update only when it changes or only update when the condition is met? The first would mean I have to call GetSimVar for every cycle (might be cheaper), the latter means it doesn't reset, which could be problematic for RTO or turn-arounds

Comment (tracernz) [reply fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts]: Use a subject to store the state, and `subject.sub((v) => SimVar.SetSimVarValue('L:A32NX_AUDIO_V1_CALLOUT', SimVarValueType.Bool, v), true);`

Comment (flogross89) [reply fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts]: good point thanks, just pushed another commit converting it into a subject

**Review by tracernz — APPROVED** [review]

**Comment by Awemeter** [comment]
Quality Assurance Tester Report

Discord Username: Awemeter
Object of testing: #9477
Aircraft: A32NX & A380X 
Tier of Testing: 1
Date: 10/11/2024 

Testing Process: 
 
Installed A380X and A32NX artifacts
Loaded into A380X
Performed a take-off and verified the V1 callout was audible slightly before V1.
Preformed a rejected takeoff after V1.
Took off once again and confirmed that the V1 callout sounded again.
Loaded into A32NX
Took off and verified the V1 callout did not sound.
Testing Results: 
Passed  

Negatives:       


Conclusions: 

LGTM
Media:

**Pull request merged**